src/main/java/de/tum/in/www1/artemis/web/rest/UserMetricsResource.java

```diff
@@ -0,0 +1,29 @@
+package de.tum.in.www1.artemis.web.rest;
+
+import org.springframework.messaging.simp.user.SimpUserRegistry;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/management")
+public class UserMetricsResource {
+
+    private final SimpUserRegistry simpUserRegistry;
+
+    public UserMetricsResource(SimpUserRegistry simpUserRegistry) {
+        this.simpUserRegistry = simpUserRegistry;
+    }
+
+    /**
+     * GET - number of current users using the application.
+     * @return number of current users using the application.
+     */
+    @GetMapping("/usermetrics")
+    @PreAuthorize("hasAnyRole('ADMIN')")
+    public int getNumberOfUsers() {
+        return this.simpUserRegistry.getUserCount();
+    }
+
+}
```

Comment:
From my understanding, this is unused and can go away?
```suggestion
```

Reply:
Yes, that is correct. I removed it now. Thank you.